github/data_source_github_organization_security_managers.go

```diff
@@ -9,6 +9,8 @@ import (
 
 func dataSourceGithubOrganizationSecurityManagers() *schema.Resource {
 	return &schema.Resource{
+		DeprecationMessage: "This data source is deprecated in favour of using the github_organization_role_team data source.",
```

Comment:
There is only a plural data source 
```suggestion
		DeprecationMessage: "This data source is deprecated in favour of using the github_organization_role_teams data source.",
```

Reply:
Good spot.